addons/dropbox/tests/test_views.py

```diff
@@ -100,8 +100,8 @@ def test_dropbox_folder_list_has_more(self, mock_list_folder, mock_list_folder_c
             if isinstance(each, MockFolderMetadata)
         ]
 
-        mock_list_folder.assert_called_once()
-        mock_list_folder_continue.assert_called_once()
+        mock_list_folder.assert_called_once_with('')
+        mock_list_folder_continue.assert_called_once_with('ZtkX9_EHj3x7PMkVuFIhwKYXEpwpLwyxp9vMKomUhllil9q7eWiAu')
```

Comment:
Nit: this change makes the assertion tightly coupled with the value set in the `MockListFolderResult`. I think `assert ...call_count == 1` is more appropriate here.